src/components/Search.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const withNavbar = (Component) => (props) => ( // export in a seperate file if redundant
```

Comment:
We have to keep the navbar separated from the search components. The search component will be passed to the Navbar as a children. The only thing we can take into account in the search comoponent is a "toggle" property, which will add the `uk-navbar-toggle` class